src/Uno.Toolkit.UI/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: InternalsVisibleTo("Uno.Toolkit.Samples")]
+[assembly: InternalsVisibleTo("Uno.Toolkit.Samples.Droid")]
+[assembly: InternalsVisibleTo("Uno.Toolkit.Samples.UWP")]
+[assembly: InternalsVisibleTo("Uno.Toolkit.Samples.WASM")]
+[assembly: InternalsVisibleTo("Uno.Toolkit.Samples.Skia.Gtk")]
```

Comment:
Is it still needed per the discussion of this morning?

Reply:
I figured I'd keep it at least for the Toolkit samples since it's useful for UI testing scenarios to make sure the margins/paddings are being applied properly